contrib/platform/src/com/sun/jna/platform/win32/Cryptui.java

```diff
@@ -24,45 +24,312 @@
+     * @param pvoid Contains information about how to do the export based on
+     * what is being exported. See above table for values, if this is non-NULL
+     * the values are displayed to the user as the default choices.
+     *
+     * @return If the function succeeds, the function returns TRUE. If the
+     * function fails, it returns FALSE. For extended error information, call
+     * GetLastError function.
+     */
+    BOOL CryptUIWizExport(DWORDByReference dwFlags, HWND hwndParent, String pwszWizardTitle,
+                          CRYPTUI_WIZ_EXPORT_INFO pExportInfo, PointerType pvoid);
+
+    /**
+     * The CRYPTUI_WIZ_IMPORT_SRC_INFO structure contains the subject to import
+     * into the CryptUIWizImport function. The subject can be a certificate, a
+     * certificate trust list (CTL), or a certificate revocation list (CRL).
+     */
+    @Structure.FieldOrder({"dwSize", "dwSubjectChoice", "certificate", "dwFlags", "pwszPassword"})
+    public static class CRYPTUI_WIZ_IMPORT_SRC_INFO extends Structure {
+        public static class ByReference extends CRYPTUI_WIZ_IMPORT_SRC_INFO implements Structure.ByReference {
+        }
+
+        public static class UNION extends Union {
+            public static class ByReference extends UNION implements Structure.ByReference {
+            }
+
+            public String pwszFileName;
+            public CERT_CONTEXT.ByReference pCertContext;
+            public CERT_CONTEXT.ByReference pCTLContext;
+            public CERT_CONTEXT.ByReference pCRLContext;
+            public HCERTSTORE hCertStore;
+        }
+
+        public DWORD cbSize;
+        public DWORDByReference dwSubjectChoice;
+        public UNION certificate;
+        public DWORD dwFlags;
+        public String pwszPassword;
+
+        public CRYPTUI_WIZ_IMPORT_SRC_INFO() {
+            super(W32APITypeMapper.DEFAULT);
+        }
+    }
+
+    /**
+     * The CRYPTUI_WIZ_EXPORT_INFO structure contains information that controls
+     * the operation of the CryptUIWizExport function.
+     */
+    @Structure.FieldOrder({"dwSize", "pwszExportFileName", "dwSubjectChoice", "certificate", "cStores", "rghStores"})
+    public static class CRYPTUI_WIZ_EXPORT_INFO extends Structure {
+        public static class ByReference extends CRYPTUI_WIZ_EXPORT_INFO implements Structure.ByReference {
+        }
+
+        public static class UNION extends Union {
+            public static class ByReference extends UNION implements Structure.ByReference {
+            }
+
+            public CERT_CONTEXT.ByReference pCertContext;
+            public CERT_CONTEXT.ByReference pCTLContext;
+            public CERT_CONTEXT.ByReference pCRLContext;
+            public HCERTSTORE hCertStore;
+        }
+
+        public DWORD dwSize;
+        public String pwszExportFileName;
+        public DWORDByReference dwSubjectChoice;
+        public UNION certificate;
+        public DWORD cStores;
+        public Pointer rghStores;
```

Comment:
The sentence "[...] array of extra stores to search for the certs in the trust chain if the chain is being exported with a cert" leads me to the question: Would this be better bound as array?